lang/en.ts

```diff
@@ -2689,11 +2693,20 @@ export const lib = {
       `Your project ${chalk.bold(projectName)} exists in ${accountIdentifier}, but has no deployed build. Projects must be successfully deployed to be developed locally. Address any build and deploy errors your project may have, then run ${uploadCommand} to upload and deploy your project.`,
     noComponents: 'There are no components in this project.',
     betaMessage: 'HubSpot projects local development',
-    learnMoreLocalDevServer: 'Learn more about the projects local dev server',
+    learnMoreLocalDevServer: uiLink(
```

Comment:
Are these translations in the en.lyaml file too? Would it make sense to remove them as a part of this, or should we wait and do that later once everything is ported?

Reply:
I've been operating under the assumption that we'd just delete the entire file at the end, so haven't been removing things from it. I don't think there's really any need to, especially since its easy to look up what we haven't ported to the new system by searching for `i18n(`